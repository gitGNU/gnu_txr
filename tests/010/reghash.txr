@(bind result @(let ((table '#H((:equal-based) 
                                ,(list "$dim" "length") ("$object" "plank")
                                ("$value" "100") ("$units" "cm"))))
                 (regsub #/$[a-z]+/ (op table)
                         "The $dim of $object is $value $units")))
