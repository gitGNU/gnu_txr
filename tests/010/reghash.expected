result="The length of plank is 100 cm"
